radio/src/gui/9X/menu_general_diaganas.cpp

```diff
@@ -38,7 +38,7 @@
 
 void menuGeneralDiagAna(uint8_t event)
 {
-#if defined(PCBSKY9X) && !defined(REVA)
+#if defined(PCBSKY9X) && !defined(REVA) && !defined(AR9X)
```

Comment:
It would be nice to introduce a new definition such as TX_CURRENT_SENSOR_AVAILABLE